Sources/StencilSwiftKit/SwiftIdentifier.swift

```diff
@@ -114,10 +114,10 @@ enum SwiftIdentifier {
   static func prefixWithUnderscoreIfNeeded(string: String,
                                            forbiddenChars exceptions: String = "") -> String {
 
-    let (head, _) = identifierCharacterSets(exceptions: exceptions)
+    guard let firstChar = string.unicodeScalars.first,
+      !string.isEmpty else { return "" }
```

Comment:
`string.isEmpty` can't ever be false if `string.unicodeScalars.first` is not `nil`, right? If `string` is empty, then `string.unocodeScalars.first` will return `nil` and already make the condition fail anyway

Reply:
You'd guess so, wouldn't you? Give it a try, for some absolutely bizarre reason, the `isEmpty` check is needed 🤷‍♂️

Reply:
Wait seriously?

isEmpty checks on characters, unicodeScalars is different, so that could make sense, maybe… but can't see an example

Do you mean that if we don't test isEmpty, the test on "" fails?

Reply:
It throws an exception (SIGINT or something I think)

Reply:
Just tested by cloning your branch and removing that `!string.isEmpty` in the condition and all test passed successfully, without SIGINT?

Reply:
Which Xcode version?

Reply:
10.0 final